RELEASE.md

```diff
@@ -0,0 +1,27 @@
+# Hdclient Release
+
+This document contains the steps to be followed to release
+the project hdclient.
+
+## Release Process
+
+To release hdclient:
+
+* Create a new branch: `git checkout -b feature/TICKET-123-release-x.y.z`
+
+* Name the tag for the repository.
+
+* Use the `npm version $tag` command with the same tag to update [package.json].
+
+* Create a PR and merge the [package.json] change.
+
+* Checkout the change in the development branch:
+  1. `git checkout development/x.y`
+  1. `git pull`
+
+* Tag the repository using the same tag:
+  1. `git tag --annotate $tag`
+  1. `git push origin $tag`
```

Comment:
isn't it `git push origin --tags` to push tags ?

Reply:
It is, the difference with `git push origin --tags` is that it will try to push all the tags that are inside your local repo.

It can happen that sometimes you may have done some manipulation on your repo and tagged it for some reason, and when you actually push the tag you want, there's a unwanted one that comes along.

So I would leave it this way, unless you would rather do the other. Both works at the end.